components/src/dynamo/planner/core/base.py

```
@@ -839,6 +847,169 @@ async def _apply_effects(self, effects: PlannerEffects) -> None:
+                continue
+            try:
+                self.connector.kube_api.patch_pod_annotation(
+                    pod.metadata.name, POWER_ANNOTATION_KEY, limit_str
+                )
+                logger.info(
+                    "Annotated pod %s with %s=%s",
+                    pod.metadata.name,
+                    POWER_ANNOTATION_KEY,
+                    limit_str,
+                )
+            except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)